Tests/Editor/TextureTests.cs

```diff
@@ -0,0 +1,58 @@
+using NUnit.Framework;
+using Unity.ProjectAuditor.Editor;
+using UnityEditor;
+using UnityEngine;
+
+namespace Unity.ProjectAuditor.EditorTests
+{
+    class TextureTests : TestFixtureBase
+    {
+        const int resolution = 1;
+        string currentPlatform = EditorUserBuildSettings.activeBuildTarget.ToString();
+        Texture textureToCompare;
+        TextureImporter textureViaImporter;
+
+        [OneTimeSetUp]
+        public void SetUp()
+        {
+            var texture = new Texture2D(resolution, resolution); //defaults: mipmaps = true & format = automatic
+            texture.SetPixel(0, 0, Random.ColorHSV());
+            texture.name = "ProceduralTextureForTest321.png";
+            texture.Apply();
+
+            var bytes = texture.EncodeToPNG();
+
+            var tempTestTexture = new TempAsset(texture.name, bytes);
+
+            var allTextures = AssetDatabase.FindAssets("t: Texture, a:assets");
```

Comment:
I don't think we need this since you already have the `texture` object, in other words `texture ` should be the same as `textureToCompare`. Seems redundant.

Also, you cannot assume the test texture to always be `allTextures[0]`

Reply:
Simply using texture, instead of textureToCompare as well.